test/models/questionnaire_test.rb

```diff
@@ -211,6 +211,120 @@ class QuestionnaireTest < ActiveSupport::TestCase
     end
   end
 
+  context "#clean_negative_special_needs" do
+    should "return nil if special_needs field is None" do
+      questionnaire = create(:questionnaire, special_needs: "NoNE")
```

Comment:
Could you add a test case for one of these having white space before & after the word?